src/hub.rs

```diff
@@ -210,7 +215,11 @@ impl Hub {
             )?)
             .await?;
 
-        inner.attributes.is_on = !inner.attributes.is_on;
+        inner.attributes.is_on = match inner.attributes.is_on {
+            None => Some(!false),
+            Some(x) => Some(!x),
+        };
```

Comment:
This doesn't seem right. If the attribute doesn't have a value for `is_on` I would suspect the device doesn't support it. Unconditionally setting it to `!false` (which we could've written as `true`) is weird because it give the impression that the device can have a value when in reality it cannot.

```suggestion
        inner.attributes.is_on = inner.attributes.is_on.map(|x| !x);
```